lib/web/resources_test.go

```diff
@@ -360,6 +360,13 @@ func TestRoleCRUD(t *testing.T) {
 
 	created := unmarshalResponse(resp.Bytes())
 
+	// Validate the role can be retrieved.
+	resp, err = pack.clt.Get(ctx, pack.clt.Endpoint("webapi", "roles", expected.GetName()), url.Values{})
+	require.NoError(t, err, "unexpected error retrieving a role")
+	assert.Equal(t, http.StatusOK, resp.Code(), "unexpected status code retrieving a role")
+	retrieved := unmarshalResponse(resp.Bytes())
+	assert.Equal(t, created, retrieved, "expected the retrieved role to be equal to the created one")
```

Comment:
Use JSONEq instead?

Reply:
I probably could, but then, this assertion would do something differently than the rest of assertions in this test. What's the reason for it? (Note that we don't compare strings here, these are unmarshalled objects.)